ivadomed/loader/utils.py

```diff
@@ -426,23 +426,45 @@ def update_filename_to_nifti(filename):
     return filename
 
 
-def dropout_input(seg_pair):
+def dropout_input(seg_pair, drop_param):
     """Applies input-level dropout: zero to all channels minus one will be randomly set to zeros. This function verifies
     if some channels are already empty. Always at least one input channel will be kept.
 
     Args:
         seg_pair (dict): Batch containing torch tensors (input and gt) and metadata.
+        drop_param (list): [p1, p2, p3], T1w, T2w and T2star have respectively p1, p2 and p3 chances to be dropped.
```

Comment:
I advise against using fixed T1w, T2w and T2star contrasts. The code needs to be as modular as possible. Eg: what if a user is only interested in microscopy and knows nothing about MRI: these acronyms won't be understood by that user. I suggest going at a higher abstract level when introducing this parameter. For example: if this is to be applied to multi channel, then name it a 'channel dropout' instead of 'T1w', etc.